workshop/content/basics/templates/pseudo-parameters/_index.md

```diff
@@ -0,0 +1,223 @@
+
+Locate the `Policies` section for `DemoRole`; replace the whole line containing `Resource: '*'` with the following:
+
+```yaml
+                Resource: !Sub 'arn:${AWS::Partition}:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${BasicParameter}'
+```
+
+Finally, add this section below to the Resources section of the `pseudo-parameters.yaml` template file. This section defines a Lambda function that uses the IAM Role you defined above with permissions to read the SSM parameter, which you also defined above. You will invoke this Lambda function to test if the Lambda function can access the SSM parameter named `dbUsername`.
+
+```yaml
+  DemoLambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Handler: index.lambda_handler
+      Role: !GetAtt DemoRole.Arn
+      Runtime: python3.8
+      Code:
+        ZipFile: |
+          import boto3
+
+          client = boto3.client('ssm')
+
+
+          def lambda_handler(event, context):
+              response = client.get_parameter(Name='dbUsername')
+              print(f'SSM dbUsername parameter value: {response["Parameter"]["Value"]}')
+
+```
+
+Save the template you have updated with content above. Next, navigate to the AWS CloudFormation [console](https://console.aws.amazon.com/cloudformation), and choose to create a stack using this template:
+* In the CloudFormation console, choose *Create stack With new resources (standard)*.
+* In *Prepare template*, choose *Template is ready*.
+* In *Template source*, choose *Upload a template file*.
+* Choose the `pseudo-parameters.yaml` template.
+* Enter a *Stack name*. For example, choose to specify `cfn-workshop-pseudo-parameters`.
+* Choose to accept the *Configure stack options* default value, and choose *Next*.
+* On the _Review_ page, scroll down to the bottom, and choose to check the box under the following *Capabilities* section: **I acknowledge that AWS CloudFormation might create IAM resources.**
+* Choose *Create stack*. You can view the progress of the stack being created in the CloudFormation console.
+* Wait until the stack creation is complete. Refresh the view in the console until you see your stack to be in the `CREATE_COMPLETE` status.
+
+![resources-png](pseudo-parameters/resources.png)
```

Comment:
The template has `DemoLambdaFunction`, and the snapshot reads `TestLambdaFunction`.  Please update the screenshot to reflect `DemoLambdaFunction`.